src/main/java/org/broadinstitute/hellbender/tools/SplitReads.java

```diff
@@ -77,6 +80,7 @@
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             doc = "The directory to output SAM/BAM/CRAM files."
     )
+    @WorkflowOutput
     public GATKPath OUTPUT_DIRECTORY;
```

Comment:
An output directory with multiple files is ok here?

Reply:
Yes, I manually tested this on one of the tools to make sure it works - I think it was this one.